docs/feature_combo.md

```diff
@@ -71,26 +92,118 @@ void process_combo_event(uint16_t combo_index, bool pressed) {
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
-
-If you're using long combos, or even longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing.
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
-
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V. But you could change this to do stuff like play sounds or change settings.
 
-## Keycodes 
+# Advanced Configuration
+These configuration settings can be set in your `config.h` file.
 
-You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+## Keycodes
+You can enable, disable and toggle the Combo feature on the fly. This is useful if you need to disable them temporarily, such as for a game. The following keycodes are avalible for use in your `keymap.c`
 
 |Keycode   |Description                      |
 |----------|---------------------------------|
 |`CMB_ON`  |Turns on Combo feature           |
 |`CMB_OFF` |Turns off Combo feature          |
 |`CMB_TOG` |Toggles Combo feature on and off |
 
+## Combo Term
+By default, the timeout for the Combos to be recognized is set to 50ms. This can be changed if accidental combo misfires are happening or if you're having difficulties pressing keys at the same time. For instance `#define COMBO_TERM 40` would set the time out period for combos to 40ms.
```

Comment:
```suggestion
By default, the timeout for the Combos to be recognized is set to 50ms. This can be changed if accidental combo misfires are happening or if you're having difficulties pressing keys at the same time. For instance, `#define COMBO_TERM 40` would set the timeout period for combos to 40ms.
```
Missing comma after “For instance”. You also sometimes write “timeout” and sometimes "time out”. It would be best to stick to one spelling. The QMK docs use “timeout” everywhere else so I'd suggest sticking to that.